.github/workflows/release.yml

```
@@ -0,0 +1,71 @@
+      - name: Build one‑dir with build.py
+        run: |
+          python -m pip install pyinstaller
+          python build.py
+
+      - name: Archive build folder
+        run: |
+          cd dist
+          zip -r "jpegcompress-${{ steps.calc.outputs.next }}.zip" jpegcompress
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)